include/swift/AST/Module.h

```diff
@@ -871,6 +871,28 @@ class ModuleDecl
   /// \sa getImportedModules
   using ImportFilter = OptionSet<ImportFilterKind>;
 
+  /// Returns an \c ImportFilter with all elements of \c ImportFilterKind.
+  constexpr static ImportFilter getImportFilterAll() {
+    return {ImportFilterKind::Exported,
+            ImportFilterKind::Default,
+            ImportFilterKind::ImplementationOnly,
+            ImportFilterKind::PackageOnly,
+            ImportFilterKind::SPIOnly,
+            ImportFilterKind::ShadowedByCrossImportOverlay};
+  }
+
+  /// Import kinds visible to the module declaring them.
+  ///
+  /// This leaves out \c ShadowedByCrossImportOverlay as even if present in
+  /// the sources it's superseded by the cross-overlay as the local import.
```

Comment:
@beccadax Can you confirm if I correctly understood the meaning of `ShadowedByCrossImportOverlay` and that we don't consider it as an import from the local module?